### DISCUSSION TITLE
Install Windows 10

### DISCUSSION THREAD

**Discussion by ncamachoh** [thegetty/quire#768] [Get Help] [answered]
Hi Quire Team,

 

I hope you're doing well.

I'm needing to install Quire in a new machine (still Windows 10 Pro) and I am consistently getting the same error message:

 

PS C:\WINDOWS\system32> npm install --global @thegetty/quire-cli

npm WARN deprecated @babel/polyfill@7.12.1: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.

npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.

 

added 532 packages, and audited 990 packages in 50s

 

132 packages are looking for funding

  run `npm fund` for details

 

35 vulnerabilities (30 moderate, 5 high)

 

To address issues that do not require attention, run:

  npm audit fix

 

Some issues need review, and may require choosing

a different dependency.

 

Run `npm audit` for details.

PS C:\WINDOWS\system32> quire --version

& : The term '-S.exe' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

At C:\Users\nataliac\AppData\Roaming\npm\quire.ps1:24 char:7

+     & "-S$exe" node --no-warnings "$basedir/node_modules/@thegetty/qu ...

+       ~~~~~~~~

    + CategoryInfo          : ObjectNotFound: (-S.exe:String) [], CommandNotFoundException

    + FullyQualifiedErrorId : CommandNotFoundException

 

PS C:\WINDOWS\system32> npm audit fix

npm ERR! code ENOLOCK

npm ERR! audit This command requires an existing lockfile.

npm ERR! audit Try creating one first with: npm i --package-lock-only

npm ERR! audit Original error: loadVirtual requires existing shrinkwrap file

 

npm ERR! A complete log of this run can be found in:

npm ERR!     C:\Users\nataliac\AppData\Local\npm-cache\_logs\2023-06-26T20_04_02_538Z-debug-0.log

 

 

Any idea why this might be happening?

**Comment by geealbers** [comment, marked as answer]
@ncamachoh sorry for the delay, are you still having this issue? If so, can you confirm what version of Node you're running? 

I think the error you're seeing is the same as issue #672 which we thought was fixed by using node 18.16.0, which was the current LTS (long term support) version at the time. LTS has been bumped to 18.16.1 but I'd expect either to work.

**Reply by ncamachoh** [reply]
Hi Greg!
Yes, updating the node version seems to have worked. Thanks so much!
